copycat/runtime/src/main/java/org/apache/kafka/copycat/cli/CopycatStandalone.java

```diff
@@ -58,9 +60,9 @@ public static void main(String[] args) throws Exception {
         workerProps = !workerPropsFile.isEmpty() ? Utils.loadProps(workerPropsFile) : new Properties();
 
         WorkerConfig workerConfig = new WorkerConfig(workerProps);
-        Worker worker = new Worker(workerConfig);
+        Worker worker = new Worker(workerConfig, new FileOffsetBackingStore());
```

Comment:
I started to write out some concerns about avoiding conflicts. This holds if you have one Kafka cluster and multiple servers -- just using some default configs could conflict easily since their offsets would be stored in the same place.

However, now that I think about it more, I'm not sure that's such a huge issue. You have this same issue using multiple standalone instances locally in order to test end-to-end with standalone mode on a local cluster (e.g. do a source connector in one copycat-standalone and a sink connector in another, all in parallel with the normal quickstart) since your offset file settings would conflict.

So I think my only argument is that it makes it hard to provide a nice experience on a shared Kafka cluster even to just run test workloads since it'd be easy to conflict with other users by accident, but I'm not convinced that's a significant issue either.